docs/getting-started/first-service.md

```diff
@@ -78,4 +78,20 @@ curl http://hello.default.127.0.0.1.nip.io
 Hello World!
 ```
 
+??? question "Are you seeing `curl: (6) Could not resolve host: hello.default.127.0.0.1.nip.io`?"
+
+    In some cases your dns server may be set up not to resolve `*.nip.io` addresses. If you encounter this problem, it can be fixed by using a different nameserver to resolve these addresses.
+
```

Comment:
```suggestion
    In some cases your DNS server may be set up not to resolve `*.nip.io` addresses. If you encounter this problem, it can be fixed by using a different nameserver to resolve these addresses.

```

Reply:
minor nit